internal/cli/atlas/accesslists/create.go

```diff
@@ -83,6 +83,10 @@ func CreateBuilder() *cobra.Command {
 		Use:   "create <entry>",
 		Short: "Create an IP access list for your project.",
 		Args:  require.ExactArgs(1),
+		Annotations: map[string]string{
+			"args":     "entry",
+			"nameDesc": "The AWS security group ID, IP address, or CIDR address of the access list entry to create.",
```

Comment:
Suggestion: I'd tweak this a little bit. Starting with AWS makes it sound like its only for AWS. 

```suggestion
			"entryDesc": "The IP address, CIDR address, or AWS security group ID of the access list entry to create.",
```